playground/mocks/data/api/v1/authn/mfa-enroll-ov-manual.json

```diff
@@ -0,0 +1,74 @@
+{
+  "stateToken": "00LUU8ZWNexHBC6i1_KNxvcS1s-xIW_hrl3ItwpUY9",
+  "expiresAt": "2022-02-03T17:32:03.000Z",
+  "status": "MFA_ENROLL_ACTIVATE",
+  "factorResult": "WAITING",
+  "_embedded": {
+    "user": {
+      "id": "00u47ptl1iWdxRIuK0g7",
+      "passwordChanged": "2022-02-02T15:11:29.000Z",
+      "profile": {
+        "login": "testuser@okta.com",
+        "firstName": "Test",
+        "lastName": "User",
+        "locale": "en_US",
+        "timeZone": "America/Los_Angeles"
+      }
+    },
+    "factor": {
+      "id": "opf47q9cwpswZcrW40g7",
+      "factorType": "push",
+      "provider": "OKTA",
+      "vendorName": "OKTA",
+      "_embedded": {
+        "activation": {
+          "expiresAt": "2022-02-03T17:27:07.000Z",
+          "factorResult": "WAITING",
+          "_links": {
+            "qrcode": {
+              "href": "https://classicpraveen.trexcloud.com/api/v1/users/00u47ptl1iWdxRIuK0g7/factors/opf47q9cwpswZcrW40g7/qr/201111Cetuu675Dg744jmJraWjFltz5nPBHWKBydJ6LY01knhjQ4cgx",
```

Comment:
can you remove these `href` urls.. and add `localhost:3000` instead ?

Reply:
done